examples/system-test/util/test-id.ts

```diff
@@ -0,0 +1,5 @@
+export const TEST_ID = {
+  welcomePage: "welcome-page",
+} as const;
+
+export type TTestId = (typeof TEST_ID)[keyof typeof TEST_ID];
```

Comment:
This isn't used it seems.

And wouldn't it be better to just have `TEST_ID` be an enum so you don't need this weird type?

Reply:
i will adjust that :) (im not a fan of enum, but we used them in this repo i agree we should keep it like so)